refresh-collaborators.py

```diff
@@ -35,7 +36,10 @@
 contributors_login_to_commit_volume = {}
 end_date = datetime.now()
 start_date = end_date - timedelta(days=365)
+repo = g.get_repo("apache/kafka")
 for commit in repo.get_commits(since=start_date, until=end_date):
+    if commit.author is None and commit.author.login is None:
```

Comment:
This should be `or` instead of `and`, both conditions will never be true together

Reply:
Correct, will change.